tdrs-backend/tdpservice/parsers/test/test_case_consistency.py

```diff
@@ -1228,15 +1228,15 @@ def test_section2_aabd_ssi_validator_fail_state_ssi(self, small_correct_file, he
                 DATE_OF_BIRTH="19970209",
                 FAMILY_AFFILIATION=1,
                 REC_AID_TOTALLY_DISABLED=2,
-                REC_SSI=2
+                REC_SSI=0
             ),
             T5Factory.create(
                 RPT_MONTH_YEAR=202010,
                 CASE_NUMBER='123',
                 DATE_OF_BIRTH="19970209",
                 FAMILY_AFFILIATION=2,  # validator only applies to fam_affil = 1; won't generate error
                 REC_AID_TOTALLY_DISABLED=2,
-                REC_SSI=2
+                REC_SSI=0
```

Comment:
Changed this to `0` to prove that the FAMILY_AFFILIATION value was excluding it from the validator and therefore only generating the single error as opposed to two errors.